ql/src/test/results/clientpositive/llap/ctas_dec_pre_scale_issue.q.out

```diff
@@ -0,0 +1,224 @@
+                    Reduce Output Operator
+                      key expressions: _col0 (type: decimal(17,7))
+                      null sort order: z
+                      sort order: +
+                      Map-reduce partition columns: _col0 (type: decimal(17,7))
+                      Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
+            Execution mode: vectorized, llap
+            LLAP IO: all inputs
+        Map 4 
+            Map Operator Tree:
+                TableScan
+                  alias: table_b
+                  filterExpr: col_dec_b is not null (type: boolean)
+                  Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
+                  Filter Operator
+                    predicate: col_dec_b is not null (type: boolean)
+                    Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
+                    Select Operator
+                      expressions: col_dec_b (type: decimal(15,5))
+                      outputColumnNames: _col0
+                      Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
+                      Reduce Output Operator
+                        key expressions: _col0 (type: decimal(17,7))
+                        null sort order: z
+                        sort order: +
+                        Map-reduce partition columns: _col0 (type: decimal(17,7))
+                        Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
+            Execution mode: vectorized, llap
+            LLAP IO: all inputs
+        Reducer 2 
+            Execution mode: llap
+            Reduce Operator Tree:
+              Merge Join Operator
+                condition map:
+                     Left Outer Join 0 to 1
+                keys:
+                  0 _col0 (type: decimal(17,7))
+                  1 _col0 (type: decimal(17,7))
+                outputColumnNames: _col0
+                Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
+                File Output Operator
+                  compressed: false
+                  Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
+                  table:
+                      input format: org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat
+                      output format: org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat
+                      serde: org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe
+                      name: default.target
+                Select Operator
+                  expressions: _col0 (type: decimal(12,7))
+                  outputColumnNames: col1
+                  Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
```

Comment:
This `Select Operator` is interesting. Although it is not involved when writing to the table it seems to have the correct data type for the column that results from the join so if we had something similar before calling the `File Output Operator` we wouldn't hit the problem in the first place.